app/common/directives/role-selector.html

```diff
@@ -40,7 +40,7 @@
                     ng-model="everyone"
                     ng-click="setAdmin()"
                 >
-                <span translate="app.specific_roles">Specific roles...</span>
+                <span translate="app.specific_roles" tabindex="0" ng-keyup="$event.keyCode == 13 && setAdmin()">Specific roles...</span>
```

Comment:
Use `===` here instead of `==`. It doesn't make much difference in this case but it is a good practice to use `===` since `==` may give unexpected results. If you want to read more about it check out these two: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness and https://dorey.github.io/JavaScript-Equality-Table/)

Reply:
Alright! I will fix this.